apps/ios/SimpleXChat/hs_init.c

```diff
@@ -39,3 +39,19 @@ void haskell_init_nse(void) {
     char **pargv = argv;
     hs_init_with_rtsopts(&argc, &pargv);
 }
+
+void haskell_init_se(void) {
```

Comment:
a copy of `haskell_init_nse`

Reply:
I don't think we need a different set of parameters here, we can just use NSE one